src/App.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js'
+
 
 const App = () => {
+  const localUser = chatMessages[0].sender
+  const remoteUser = chatMessages[1].sender
```

Comment:
What if textMessage[0] and textMessage[1] were both messages from the same person? How else could you find a way to get the names of the senders?